
<launch> 
  <node pkg="planning_environment" type="clear_known_objects" respawn="false" output="screen">
    <remap from="robot_description" to="robotdesc/pr2" />
    <param name="fixed_frame" type="string" value="/base_link" />
    <remap from="cloud_in" to="full_cloud_annotated" />
    <remap from="cloud_out" to="full_cloud_without_known_objects" />
  </node>
</launch>
